solr/contrib/ltr/src/java/org/apache/solr/ltr/LTRScoringQuery.java

```diff
@@ -210,7 +208,7 @@ public ModelWeight createWeight(IndexSearcher searcher, boolean needsScores, flo
     final FeatureWeight[] modelFeaturesWeights = new FeatureWeight[modelFeatSize];
     List<FeatureWeight > featureWeights = new ArrayList<>(features.size());
     
-    if(ltrThreadMgr.getMaxThreads() <= 1 || ltrThreadMgr.getMaxQueryThreads() <= 1){
+    if (ltrThreadMgr == null) {
```

Comment:
Could you clarify, `validate()` should reject thread managers which specify either `maxQueryThreads` or `maxThreads` to be 1?